Sprint-2/implement/bmi.js

```diff
@@ -13,3 +13,67 @@
 // Given someone's weight in kg and height in metres
 // Then when we call this function with the weight and height
 // It should return their Body Mass Index to 1 decimal place
+
+________________________________________SOLUTION_________________________________________
+
+// Implementation:
+
+/**
+ * Calculates the Body Mass Index (BMI) given weight in kg and height in meters.
+ *
+ * @param {number} weight - The weight of the person in kilograms.
+ * @param {number} height - The height of the person in meters.
+ * @returns {number} The BMI value rounded to 1 decimal place.
+ */
+function calculateBMI(weight, height) {
+    if (height <= 0 || weight <= 0) {
+      throw new Error("Height and weight must be positive values.");
+    }
+  
+    // Step 1: Square the height
+    const heightSquared = height ** 2;
```

Comment:
why ` **` and not `height * height` or `Math.pow(height, 2)` what's the difference?